### PR TITLE
Add v20 changelog docs for PR #15030

### PR DESCRIPTION
**Pull request by timvaillancourt**
## Description

This PR adds a changelog entry to v20 for https://github.com/vitessio/vitess/pull/15030 now that a changelog subdir exists

## Related Issue(s)

https://github.com/vitessio/vitess/pull/15030

## Checklist

-   [ ] "Backport to:" labels have been added if this change should be back-ported to release branches
-   [ ] If this change is to be back-ported to previous releases, a justification is included in the PR description
-   [ ] Tests were added or are not required
-   [ ] Did the new or modified tests pass consistently locally and on CI?
-   [ ] Documentation was added or is not required

### CODE REVIEW

**Review by frouioui — APPROVED** [review]
LGTM! Thanks for having kept track of this.

**Review by systay — APPROVED** [review]

**Comment by systay** [comment]
@timvaillancourt seems like you missed signing that last commit. Our DCO check is angry.

@frouioui the release notes failing is not very informative. do you if we could we make these checks a bit more verbose if they fail?

**Comment by deepthi** [comment]
oh @timvaillancourt the DCO check is failing. You'll need to amend the commit with a signoff and force-push it.

**Comment by deepthi** [comment]
@frouioui can you look at the static check failure? It's from `Check changelog` so I'm not sure why it's complaining about go version.

File: changelog/20.0/20.0.0/summary.md [hunk 1]
```diff
@@ -72,3 +73,7 @@ VTTablet exposes two new counter stats:
 
  * `QueryCacheHits`: Query engine query cache hits
  * `QueryCacheMisses`: Query engine query cache misses
+
+### <a id="sighup-reload-of-grpc-client-auth-creds"/>`SIGHUP` reload of gRPC client static auth creds
+
+The internal gRPC client now caches the static auth credentials and supports reloading via the `SIGHUP` signal. Previous to v20 the credentials were not cached and loaded from disk on the fly.
```
[the remark this answers is not in the document]
Comment (timvaillancourt) [reply changelog/20.0/20.0.0/summary.md]: @deepthi that wording is more clear, thanks! Updated 👍

**Review by frouioui — APPROVED** [review]

**Comment by frouioui** [comment]
> can you look at the static check failure? It's from `Check changelog` so I'm not sure why it's complaining about go version.

@deepthi done, just had to merge `main`.

**Pull request merged**